editor/plugins/gdextension_export_plugin.h

```diff
@@ -117,23 +117,23 @@ void GDExtensionExportPlugin::_export_file(const String &p_path, const String &p
 
 			if (p_features.has("ios") && (library_path.ends_with(".a") || library_path.ends_with(".xcframework"))) {
```

Comment:
Maybe all apple embedded platforms should enter this branch. For consistency, all apple embedded platforms should additionally declare a common platform feature "apple_embedded".

```suggestion
			if (p_features.has("apple_embedded") && (library_path.ends_with(".a") || library_path.ends_with(".xcframework"))) {
```